lib/Runtime/PlatformAgnostic/Platform/Linux/ArraysUtility.cpp

```diff
@@ -0,0 +1,81 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Common.h"
+#include "ChakraPlatform.h"
+#include <locale.h>
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+    #define BufSize 256
+
+    template<class T, class N = T>
+    void ByteCopy(const T *from, N *to, const size_t max_length)
+    {
+        size_t length = 0;
+        for (; length < max_length && (to[length] = static_cast<N>(from[length])) != 0;
+            length++);
+        if (length == max_length)
+        {
+            to[length - 1] = N(0);
+        }
+    }
+
+    Utility::ArraysLocale::ArraysLocale()
+    {
+
+        commaSeparator = WCHAR(',');
+        semicolonSeparator = WCHAR(';');
```

Comment:
We usually use `_u('')` for wide string literals. Also, why do these have to be class members? It looks like theyre only used in this method, and if nothing else, they could be static since it doesn't seem like they ever change.